src/main/java/uk/gov/hmcts/darts/usermanagement/service/impl/SecurityGroupServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.darts.usermanagement.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.common.entity.SecurityGroupEntity;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.repository.SecurityGroupRepository;
+import uk.gov.hmcts.darts.common.repository.SecurityRoleRepository;
+import uk.gov.hmcts.darts.usermanagement.exception.UserManagementError;
+import uk.gov.hmcts.darts.usermanagement.mapper.impl.SecurityGroupMapper;
+import uk.gov.hmcts.darts.usermanagement.model.SecurityGroup;
+import uk.gov.hmcts.darts.usermanagement.model.SecurityGroupWithIdAndRole;
+import uk.gov.hmcts.darts.usermanagement.service.SecurityGroupService;
+
+import java.util.Collections;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class SecurityGroupServiceImpl implements SecurityGroupService {
+
+    private final SecurityGroupRepository securityGroupRepository;
+    private final SecurityRoleRepository roleRepository;
```

Comment:
Should we call this variable `securityRoleRepository` in keep with the one above?